src/modules/integrator-guide/pages/automation/workflows/examples/template-rendering.adoc

```diff
@@ -0,0 +1,164 @@
+= Template rendering
+:attachment-path: ../../_attachments/workflow/
+
+In this example, we'll take a look at how to render different templates that were made in our xref:templates/index.adoc[template system].
+
+To start, create a template of type HTML in our templating system located in the admin area.
+
+== HTML
+
+Create a workflow similar to the one below. Make sure to look at the step details of steps (2) and (3).
+
+.The screenshot outlines the workflow that renders an HTML template and sends it as an email.
+image::workflow/example-template-rendering/html-workflow.png[role="data-zoomable"]
+
+The link:{attachment-path}template_rendering_html.json[source code] for the workflow example.
+
+
+****
```

Comment:
You may omit this bit in the future (the whole step breakdown); I'm working on something way better

Reply:
We need this until we have the better thing, since otherwise you don't know how steps are defined

Reply:
My rationale behind not having to include it in is the exported JSON; you can leave it and can keep on writing these, but it's no longer required.